shell/platform/windows/client_wrapper/flutter_view_controller.cc

```diff
@@ -0,0 +1,65 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "include/flutter/flutter_view_controller.h"
+
+#include <algorithm>
+#include <iostream>
+
+namespace flutter {
+
+FlutterViewController::FlutterViewController(
+    const std::string& icu_data_path)
+    : icu_data_path_(icu_data_path) {
+}
+
+FlutterViewController::~FlutterViewController() {
+  if (controller_) {
+    FlutterDesktopDestroyView(controller_);
+  }
+}
+
+FlutterView FlutterViewController::CreateFlutterView(
+    int width,
+    int height,
+    const std::string& assets_path,
+    const std::vector<std::string>& arguments) {
+
+  if (controller_) {
+    std::cerr << "Only one Flutter window can exist at a time." << std::endl;
```

Comment:
s/window/view/ here and in the other error strings

Reply:
Fixed in next commit